### PR TITLE
Add NET_BIND_SERVICE capability

### PR DESCRIPTION
**Pull request by mrueg**
If we add this one, probably a lot of addons could drop NET_ADMIN if they need to bind a well-known port.

### CODE REVIEW

File: src/config.schema.json [hunk 1]
```diff
@@ -212,7 +212,8 @@
           "DAC_READ_SEARCH",
           "IPC_LOCK",
           "NET_ADMIN",
-          "NET_RAW",
+          "NET_BIND_SERVICE",
+	  "NET_RAW",
```
Comment (frenck) [inline src/config.schema.json:216]: Small styling issue

```suggestion
          "NET_BIND_SERVICE",
          "NET_RAW",
```

**Review by frenck — CHANGES_REQUESTED** [review]
I'm not aware of the Home Assistant Supervisor supporting this?

I might be wrong, could you please point me to a resource (or sourcecode) that shows the existence of this flag for add-ons?

Thanks! 👍 

../Frenck

**Comment by mrueg** [comment]
> I'm not aware of the Home Assistant Supervisor supporting this?
> 
> I might be wrong, could you please point me to a resource (or sourcecode) that shows the existence of this flag for add-ons?
> 
> Thanks! 👍
> 
> ../Frenck
Thanks for clarifying! I thought the supervisor just passes it through to docker and I wasn't aware that it gets filtered. 
What would we need to enable this? Is adding it to https://github.com/home-assistant/supervisor/blob/f150a19c0f77ce3f5223ed563420b38128d2307e/supervisor/docker/const.py (and other occurrences where capabilities are passed to docker) enough?

**Comment by frenck** [comment]
> What would we need to enable this? 

Get it added upstream :)

I'm going to close this PR on this end, as it turns out to be invalid at this point.

../Frenck

**Pull request closed without merging**